libs/template/themes/src/fulfillment.theme.ts

```diff
@@ -0,0 +1,12 @@
+import { Theme } from '@spryker-oryx/core';
+import { defaultBreakpoints } from '@spryker-oryx/themes/breakpoints';
+import { fulfillmentIcons } from '@spryker-oryx/themes/icons';
+
+// TODO: This should be dropped after HRZ-2239
```

Comment:
Do we need this comment?

Reply:
Nope, I guess, I've just restored the removed theme, but I believe after cleaning of the mess with icon for diff themes we will have only 2 of them, and this one will be removed